ppr-ui/src/components/tombstone/TombstoneDischarge.vue

```diff
@@ -18,7 +18,7 @@
             <span class="float-right">Registration Status: </span>
           </v-col>
           <v-col class="pl-3" cols="6">
-            {{ statusType }}
+            {{ statusType[0] + statusType.toLowerCase().slice(1) }}
```

Comment:
Does this affect PPR?

Reply:
This was just UXA for previous ticket 15934. Made sure in that ticket no changes were made to ppr

Reply:
Thanks!